### DISCUSSION TITLE
RestDW funciona em 64bits?

### DISCUSSION THREAD

**Discussion by jerrivaldo** [OpenSourceCommunityBrasil/REST-DataWare#331] [General]
Fiz um programa que usa o seguinte comando: RESTDWIdClientREST1.Get(URL, Nil, Response);
Compilando em Delphi 32bits funciona normalmente , mas se eu compilar em 64bits dá o seguinte erro: "Connection Closed Gracefully." e não funciona.

**Comment by mobius1qwe** [comment]
Funciona normalmente em 64bits. Eu mesmo não compilo nada em 32 e tudo funciona.

Precisa garantir que seu servidor seja 64bits. Não existe servidor 32bits mais, claro, considerando que a única coisa que mudou entre um teste e outro foi só compilação. Se mais coisa mudou aí é problema do código.

**Comment by jerrivaldo** [comment]
O problema era a versão do OpenSSL. Atualizei o OpenSSL para openssl-1.0.2q-x64_86-win64 e funcionou perfeitamente. Consegui essa versão do OpenSSL em http://indy.fulgan.com/SSL/

**Comment by mobius1qwe** [comment]
Na pasta extras da versão atual também tem as .dlls tanto 32 quanto 64.
